package.json

```
@@ -44,7 +44,7 @@
     "seed": "tsx scripts/seed-local-env.ts",
     "start": "pnpm run local:setup",
     "test": "vitest",
-    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run",
+    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run --browser chrome",
```

```suggestion
    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run",
```

TODO @n1ru4l  need to check whether it runs without chrome